apps/fishing-map/data/default-workspaces/workspace.development.ts

```diff
@@ -149,6 +150,13 @@ const workspace: Workspace<WorkspaceState> = {
       },
       dataviewId: HIGH_SEAS_DATAVIEW_ID,
     },
+    {
+      id: 'context-layer-graticules',
+      config: {
+        visible: false,
+      },
+      dataviewId: GRATICULES_DATAVIEW_ID,
```

Comment:
Would be nice to have it in the [documentation](https://github.com/GlobalFishingWatch/frontend/blob/develop/apps/fishing-map/data/dataviews.md#L15)